components/toolbar/test/index.js

```diff
@@ -33,7 +33,6 @@ describe( 'Toolbar', () => {
 			];
 			const toolbar = shallow( <Toolbar controls={ controls } /> );
 			const listItem = toolbar.find( 'IconButton' );
```

Comment:
I found this [jest-enzyme](https://github.com/blainekasten/enzyme-matchers) yesterday. It might simplify testing components. We can also experiment with snaphsots to minimize the number of assertions where we validate HTML, which should be an implementation detail that is more likely to change.